tests/components/webostv/test_media_player.py

```diff
@@ -52,6 +56,11 @@ async def setup_webostv(hass):
     await hass.async_block_till_done()
 
 
+def cleanup_webostv(hass):
+    """Test cleanup, remove the config file."""
+    os.remove(hass.config.path(WEBOSTV_CONFIG_FILE))
```

Comment:
Make this a pytest fixture so it's also executed when the test fails.
```suggestion
@pytest.fixture
def cleanup_config(hass):
    """Test cleanup, remove the config file."""
    yield
    os.remove(hass.config.path(WEBOSTV_CONFIG_FILE))
```